includes/class-webhooks.php

```diff
@@ -20,20 +20,44 @@ class Webhooks {
 	 */
 	protected $capability = 'manage_options';
 
+	/**
+	 * If this site is a duplicate (staging or dev clone) site,
+	 * don't handle webhooks which communicate with external services.
+	 *
+	 * @var boolean
+	 */
+	protected $is_duplicate_site = false;
+
 	/**
 	 * Constructor.
 	 */
 	public function __construct() {
 		$this->salesforce_settings = Salesforce::get_salesforce_settings();
 
 		add_action( 'rest_api_init', [ $this, 'register_api_endpoints' ] );
+		add_filter( 'woocommerce_subscriptions_is_duplicate_site', [ $this, 'check_duplicate_site_status' ] );
```

Comment:
Might be worth bumping the priority on this to `999` or something. That way we're still good if something else hooks into this filter and returns that it's a duplicate site

Reply:
Good idea! Updated in 30f6525f.